packages/form-builder/src/components/canvas/DesignMode.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {useEffect, useState} from 'react';
+import {Storage} from '../../common';
+import IframeResizer from 'iframe-resizer-react';
+
+const DesignMode = ({blocks}) => {
```

Comment:
Can we actually rename this file and component to `DesignPreview`.  We moved away from the term _design mode_ so just want to make sure we don't confuse ourselves.

Reply:
Resolved by https://github.com/impress-org/givewp-next-gen/pull/76/commits/a3a7c6f36c65a3269b3bf7b05843ae9bb78ee83a.